src/Bild/Hook/ConfigurationHook.php

```diff
@@ -1,43 +0,0 @@
-<?php
-
-namespace Bild\Hook;
-
-use Consolidation\AnnotatedCommand\Hooks\InitializeHookInterface;
-use Consolidation\AnnotatedCommand\AnnotationData;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Yaml\Parser;
-
-//use Symfony\Component\Config;
-
-class ConfigurationHook implements InitializeHookInterface {
-
-  /**
-   * @inheritdoc
-   */
-  public function initialize(InputInterface $input, AnnotationData $data) {
-    //TODO - Explore Symfony Config.
-    /*$configDirectories = array(__DIR__);
-    $locator = new Config\FileLocator($configDirectories);
-    $config = $locator->locate('bild.yml', NULL, TRUE);*/
-
-    //TODO - Explore "state" of application to set config not as option.
-
-    // Load config from current runtime directory.
-    $project_config = getcwd().'/bild.yml';
-    if (file_exists($project_config)) {
-      //TODO - Consider config validation.
-
-      $parser = new Parser();
-      $project_config = $parser->parse(file_get_contents($project_config));
```

Comment:
I accidentally took this part out. Just put it back in, in your main hook, then feed this into the Robo $config object's `extend()` method.